cms/io/triggeredservice.py

```diff
@@ -154,6 +154,19 @@ def run(self):
                         "Unexpected error when executing operation `%s'.",
                         to_execute[0].item, exc_info=True)
 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/s-maggiolo/cms/file/2232352172/issues/source?bid=3167167&fileBranchId=3167180#l157)